packages/components/psammead-brand/src/index.stories.jsx

```diff
@@ -1,9 +1,10 @@
 import React from 'react';
 import { storiesOf } from '@storybook/react';
-import { withReadme } from 'storybook-readme';
 import Readme from '../README.md';
```

Comment:
```suggestion
import notes from '../README.md';
```
By changing the name of the import, we could tidy up the notes declaration below.

Reply:
Done :)